src/components/Button.js

```diff
@@ -21,4 +20,6 @@ export const Button = (props) => {
 
 Button.propTypes = {
   name: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
+  wide: PropTypes.string.isRequired,
```

Comment:
- [ ] I got also another console error here:
![Screenshot from 2021-01-14 22-23-47](https://user-images.githubusercontent.com/4246637/104645058-3d287e80-56b7-11eb-9c8a-7d3b912afd28.png)

Let's fix this.